index.html

```diff
@@ -18,68 +21,122 @@
     <link rel='apple-touch-icon' sizes='144x144' href='/images/favicons/apple-icon-144x144.png'>
     <link rel='apple-touch-icon' sizes='152x152' href='/images/favicons/apple-icon-152x152.png'>
     <link rel='apple-touch-icon' sizes='180x180' href='/images/favicons/apple-icon-180x180.png'>
-    <link rel='icon' type='image/png', sizes='192x192', href='/images/favicons/android-icon-192x192.png'>
-    <link rel='icon' type='image/png', sizes='32x32', href='/images/favicons/favicon-32x32.png'>
-    <link rel='icon' type='image/png', sizes='96x96', href='/images/favicons/favicon-96x96.png'>
-    <link rel='icon' type='image/png', sizes='16x16', href='/images/favicons/favicon-16x16.png'>
-
+    <link rel='icon' type='image/png' , sizes='192x192' , href='/images/favicons/android-icon-192x192.png'>
+    <link rel='icon' type='image/png' , sizes='32x32' , href='/images/favicons/favicon-32x32.png'>
+    <link rel='icon' type='image/png' , sizes='96x96' , href='/images/favicons/favicon-96x96.png'>
+    <link rel='icon' type='image/png' , sizes='16x16' , href='/images/favicons/favicon-16x16.png'>
```

Comment:
Cool that you add Android icons!

The commas in the `<link>` attributes list should not be there. That is, attributes are whitespace-separated, no need for a comma.

Reply:
Thank you for the comment. These icons are from the project template but good they included. We will remove the commas :)